src/main/features/dashboard/views/defendant.njk

```diff
@@ -14,28 +14,38 @@
     <div class="column-two-thirds">
       <h2 class="heading-medium">{{ t('Claim status') }}</h2>
 
-      {{ responseStatusForDefendant(claim, DashboardUrlHelper.getStatusUrl(claim), isAfter4pm) }}
-
       {% if featureToggles.offer and claim.status === ClaimStatus.OFFER_ACCEPTED %}
-        <p>
-          <a href="{{ OfferPaths.countersignAgreementPage.evaluateUri({ externalId: claim.externalId }) }}">
-            {{ t('Sign the agreement') }}
-          </a>
-        </p>
-      {% elif featureToggles.offer and not claim.settlementReachedAt  %}
-        <h2 class="heading-medium">{{ t('More options') }}</h2>
-        <h2 class="heading-medium">{{ t('Settle out of court') }}</h2>
-        <p>{{ t('You can still <a href="%s">make an offer</a> to settle the claim out of court. You will avoid a hearing and a County Court Judgment against you.',
-                { postProcess: 'sprintf',
-                  sprintf: [ OfferPaths.offerPage.evaluateUri({ externalId: claim.externalId }) ]
-                }) | safe
-            }}
-        </p>
+        {% for defendantClaimState in claim.stateHistory %}
```

Comment:
The same comment here, claim state is the same for claimant and defendant, please rename variable to be just `claimState` or `state`